lambdas/section-manager-lambda/src/utils.ts

```diff
@@ -129,9 +147,49 @@ export const processSqsSectionData = async (
       // continue to next item
     }
   }
+  // Remove "old" SectionItems not present in ML payload AFTER new ones are created
+  if (sectionItemsToRemove.length > 0) {
+    for (const item of sectionItemsToRemove) {
+      try {
+        await removeSectionItem(config.adminApiEndpoint, graphHeaders, {
+          externalId: item.externalId,
+          deactivateReasons: [SectionItemRemovalReason.OTHER],
```

Comment:
OTHER reason seems to be the only fitting --- or should we add a new one for ML?

Reply:
I made my comment before reading Jonathan's. Agreed with Jonathan.

Reply:
"ML" is not a reason so we would need to add it in this case to the enum